pkgs/tools/security/fail2ban/default.nix

```diff
@@ -47,7 +47,7 @@ python3.pkgs.buildPythonApplication rec {
     in
     ''
       # see https://github.com/NixOS/nixpkgs/issues/4968
-      rm -r ${sitePackages}/etc ${sitePackages}/usr
+      rm -rf ${sitePackages}/etc ${sitePackages}/usr
```

Comment:
If one of those directories does not exit on darwin we should rather make this conditional rather than adding -f and ignore the failure.